### DISCUSSION TITLE
Why does help output of pyclass say that its from buitins.<class_name> rather than the <module_name>.<class_name>

### DISCUSSION THREAD

**Discussion by psvri** [PyO3/pyo3#3201] [Questions] [answered]
Hello,

Why does the print message in https://docs.rs/pyo3/latest/pyo3/types/struct.PyModule.html#method.add_class say that its from builtins as seen in output `Foo is <class 'builtins.Foo'>` instead of  `Foo is <class 'my_module.Foo'>`

**Comment by davidhewitt** [comment]
See #1820

**Reply by psvri** [reply, marked as answer]
Thanks for the information and looking into it
